### PR TITLE
Updating people and place icons

### PR DESCRIPTION
**Pull request by Jodge**
# Description

Update place icons - minor visual changes
Updating some **png** images files to **svg** _(scaling well to any size without losing quality)_

medic/medic-webapp#4895

# Review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://github.com/medic/medic-docs/blob/master/development/style-guide.md), documented if necessary.
- [ ] Documented: Announced in Changes.md in plain English. Configuration and user documentation on [medic-docs](https://github.com/medic/medic-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in Changes.md.

### CODE REVIEW

File: config/standard/resources.json [hunk 1]
```diff
@@ -1,26 +1,25 @@
...
    "immunization": "icon-healthcare-immunization@2x.png",
    "infant": "icon-people-baby@2x.png",
    "mother-child": "icon-people-woman-baby@2x.png",
-   "new-person": "icon-people-person-general@2x.png",
+   "new-person": "icon-people-person-general@2x.svg",
    "nurse": "icon-people-nurse@2x.png",
    "off": "icon-messages-off@2x.png",
    "on": "icon-messages-on@2x.png",
-   "person": "icon-people-person-general@2x.png",
+   "person": "icon-people-person-general@2x.svg",
```
Comment (garethbowen) [inline config/standard/resources.json]: I think we should remove the `2x` suffix from these file names - it doesn't make sense with svgs as they scale to any size.

File: config/standard/resources.json [hunk 2]
```diff
@@ -1,26 +1,25 @@
...
    "pregnancy-1": "icon-people-woman-pregnant@2x.png",
    "pregnancy-4": "icon-people-woman-pregnant@2x.png",
    "risk": "icon-healthcare-warning@2x.png",
    "chw": "icon-people-CHW-female@2x.png",
    "medic-clinic": "medic-clinic.svg",
    "medic-health-center": "medic-health-center.svg",
-   "medic-district-hospital": "medic-district-hospital.svg",
-   "medic-person": "medic-person.svg"
+   "medic-district-hospital": "medic-district-hospital.svg"
```
Comment (garethbowen) [inline config/standard/resources.json]: I see you've deleted "medic-person" - are you sure it's never used?

Comment (Jodge) [reply config/standard/resources.json]: Oops!

File: config/standard/resources.json (hunk 1 above)
Comment (Jodge) [reply config/standard/resources.json]: Ok

**Pull request merged**